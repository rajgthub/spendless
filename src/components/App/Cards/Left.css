.heading{
    color: rgb(70, 207, 132);
    font-size: 2.5rem;
    font-weight: bold;
}