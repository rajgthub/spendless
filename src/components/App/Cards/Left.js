import React from "react";
import "./Left.css";
import AnualIncomes from "../AnualIncomes";
import MonthlySpending from "../MonthlySpending";
const Left = () => (
  <div className="card_left">
    <div className="title">your income & spend</div>
    <section className="card_body">
      <h1 className="heading">Anual incomes</h1>
      <AnualIncomes />
      <h1 className="heading">Monthly spending</h1>
      <MonthlySpending />
    </section>
  </div>
);
export default Left;
