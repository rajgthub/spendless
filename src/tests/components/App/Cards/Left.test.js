import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Left from '../../../../components/App/Cards/Left';
test("should render Header ", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Left />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
