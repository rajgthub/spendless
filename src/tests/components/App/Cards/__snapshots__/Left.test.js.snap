// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header  1`] = `
<div
  className="card_left"
>
  <div
    className="title"
  >
    your income & spend
  </div>
  <section
    className="card_body"
  >
    <h1
      className="heading"
    >
      Anual incomes
    </h1>
    <Connect(AnualIncomes) />
    <h1
      className="heading"
    >
      Monthly spending
    </h1>
    <Connect(MonthlySpending) />
  </section>
</div>
`;
